src/translations/en.json

```diff
@@ -1079,6 +1080,42 @@
             "dialog": {
               "selected_stat_intro": "Select the entity that represents the device energy usage."
             }
+          },
+          "validation": {
```

Comment:
@matthiasdebaat here are all the validation errors that we can show with their title/description.